src/main/java/com/coherent/training/api/kapitsa/clients/Users.java

```diff
@@ -0,0 +1,73 @@
+package com.coherent.training.api.kapitsa.clients;
+
+import com.coherent.training.api.kapitsa.util.plainobjects.User;
+import lombok.SneakyThrows;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static com.coherent.training.api.kapitsa.providers.Scope.READ;
+import static com.coherent.training.api.kapitsa.providers.Scope.WRITE;
+import static com.coherent.training.api.kapitsa.providers.UrlProvider.USERS;
+
+public class Users extends BaseClient{
+    private static final String USERS_ENDPOINT = USERS.getEndpoint();
+    private static final org.slf4j.Logger logger = LoggerFactory.getLogger(Users.class.getSimpleName());
+
+    public Users(CloseableHttpClient client) {
+        super(client);
+    }
+
+    @SneakyThrows
+    public User addUser(String userJson){
+        StringEntity entity = new StringEntity(userJson);
+
+        baseClient = builder.setRequest(USERS_ENDPOINT, POST, setHeadersMap(WRITE), entity).build();
+
+        User requestBody = baseClient.getRequestBody(User.class);
+
+        logger.info("Request: {}", baseClient.getRequest() + "\n" + requestBody);
+
+        response = baseClient.executeRequest();
+
+        logger.info("Response: {}", response);
```

Comment:
let's log response code, endpoint, headers and body, same for request (in base client or interceptor)